src/core/lib/address_utils/parse_address.cc

```diff
@@ -103,8 +103,17 @@ grpc_error_handle UnixSockaddrPopulate(absl::string_view path,
         "Path name should not have more than ", maxlen, " characters"));
   }
   un->sun_family = AF_UNIX;
-  path.copy(un->sun_path, path.size());
-  un->sun_path[path.size()] = '\0';
+  int offset_start = 0;
+  if (absl::StartsWith(path, "///")) {
+    std::cout << "offset" << std::endl;
+    offset_start = 2;
+  } else if (absl::StartsWith(path, "//")) {
+    return GRPC_ERROR_CREATE(absl::StrCat(
+        "Absolute path should start with either a single slash caracter, or triple slash characters"));
+  }
+  path.copy(un->sun_path, path.size() - offset_start, offset_start);
+  un->sun_path[path.size() - offset_start] = '\0';
+  std::cout << "un->sun_path " << un->sun_path <<  std::endl;
```

Comment:
Logging in grpc is done with `gpr_log(GPR_DEBUG, "un->sun_path: %s", un->sun_path);`

Reply:
Fixed in [21d840d](https://github.com/grpc/grpc/pull/36036/commits/21d840dabdb9e26a4b49ddab066c3552a7118516)